packages/geojsone/geojsone.0.1.0/opam

```diff
@@ -30,7 +31,7 @@ build: [
     jobs
     "--promote-install-files=false"
     "@install"
-    "@runtest" {with-test}
+    "@runtest" {with-test & ocaml:version < "5.0"}
```

Comment:
It doesn't make much sense to disable on ocaml5 only given eio is already ocaml5 only. However the breakage happened with eio 0.7
```suggestion
    "@runtest" {with-test & eio:version < "0.7"}
```